README.md

```diff
@@ -201,3 +201,11 @@ CFEngine 3.15.3 (Enterprise) was successfully installed on 'localhost'
 
 Feel free to open pull requests to expand this documentation, add features or fix problems.
 You can also pick up an existing task or file an issue in [our bug tracker](https://tracker.mender.io/issues/?filter=11711).
+
+## Development
+
+To install `cf-remote` so that it reflects any changes in this source directory use:
```

Comment:
Does this do the same as `pip install .`?

Reply:
Ah, that would be `pip install -e .` I think. Which may have the advantage of letting pip know you installed it. I'll change it. Though in testing I see that using the "old way" also seems recognized by pip.

```-e, --editable <path/url>
Install a project in editable mode (i.e. setuptools “develop mode”) from a local project path or a VCS url.```